libs/evm-protocols/src/common-protocol/chainConfig.ts

```diff
@@ -8,6 +8,8 @@ export enum ValidChains {
   Optimism = 10,
   Mainnet = 1,
   Arbitrum = 42161,
+  BSC = 56,
+  SKALE = 974399131,
```

Comment:
This chain id is for `SKALE Calypso Hub Testnet`. We should name this something more accurate like: `SKALE_CALYPSO_TESTNET`

Reply:
Should be good to use SKALE_TEST as we're only going to use this for testnet then replace with mainnet. Also updated event registry